server.js

```diff
@@ -21,8 +21,8 @@
 const http = require('http');
 const { MongoClient } = require('mongodb');

```

Comment:
Consider using a more descriptive variable name instead of 'HOSTNAME' for better clarity.